infra/src/test/kotlin/fi/fta/geoviite/infra/dataImport/CsvParsingTest.kt

```diff
@@ -374,15 +370,15 @@ class CsvParsingTest {
         assertEquals(expected, segments)
     }
 
-    @Disabled // TODO: clean up or fix old version test
     @Test
     fun segmentingRangesWorksWithSinglePointSwitchAtStart() {
```

Comment:
Tästä alkaa kiinnostava osuus, eli oikea toiminnallinen muutos. Uusin versio ei yhdistä 1-piste-vaihdetta osaksi lähintä segmenttiä vaan generoi sille oman segmentin. 3 testiä tästä eteenpäin näyttää eron.